src/data_import/data_import.py

```diff
@@ -111,11 +113,36 @@ def get_dataset_path(filename: Union[str, Path]) -> Path:
 
     Examples
     --------
-    >>> from edgePy.io import get_dataset_path
+    >>> from src.data_import.data_import import get_dataset_path
     >>> str(get_dataset_path("GSE49712_HTSeq.txt.gz"))  # doctest:+ELLIPSIS
-    '.../edgePy/data/GSE49712_HTSeq.txt.gz'
+    '.../src/data/GSE49712_HTSeq.txt.gz'
 
     """
-    import edgePy
-    directory = Path(edgePy.__file__).expanduser().resolve().parent
+    import src
+    directory = Path(src.__file__).expanduser().resolve().parent
     return directory / 'data' / filename
+
+
+def create_DGEList(sample_list: List[str],
```

Comment:
Not documented. What is this used for (need docstring) and why is the code not important enough for existing as a classmethod upon the DGEList itself? Our code will be more modular and far easier to maintain if this exists as a classmethod.